#!/usr/bin/env python
PACKAGE = "point_cloud_test"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("int_param",int_t,0,"int parameter",1,0,10)
gen.add("inflation_radius",double_t,0,"double parameter",0.6,0.0,2.0)
gen.add("cluster_threshold_slope",double_t,0,"double parameter",0.15,0.0,1.0)
gen.add("similarity_coefficient_1",double_t,0,"double parameter",1,0,100)
gen.add("variance_of_points_num",double_t,0,"double parameter",100,0,1000)
gen.add("similarity_coefficient_2",double_t,0,"double parameter",1,0,100)
gen.add("variance_of_distance",double_t,0,"double parameter",10,0,100)

gen.add("predicted_variance",double_t,0,"double parameter",1,0,10)
gen.add("measure_variance",double_t,0,"double parameter",1,0,10)

gen.add("inflation_layer_radius",double_t,0,"double parameter",0.5,0,2)
gen.add("position_offset",double_t,0,"double parameter",0.5,0,3)

exit(gen.generate(PACKAGE,"point_cloud_test","point_cloud_processe_"))
